visidata/features/errors_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class ErrorsGuide(GuideSheet):
+    guide_text='''# A guide to errors in VisiData
+
+There are two severity levels VisiData will use when communicating something has gone wrong.
+If in performing an operation it reaches a [:166]warning[/], the operation will continue.
+With [:202]errors[/], operations will stop.
+
+[:202]Errors[/] and/or [:166]warnings[/] can manifest in different ways. How they appear determines how they are folllowed up on.
+
+1. [:202]a phrase all in lowercase[/] [:166]appears in the sidebar[/]
+This is a hand-written message written for a known potential scenario. An example is *"invalid regex"* when the user passes an invalid regex string to a VisiData command.
+For more information on what condition triggered the message `git grep` for it within the VisiData repo.
+
+2. The first line of a stacktrace for an [:202]Exception[/] appears in the sidebar.
```

Comment:
"A Python Exception like RuntimeError appears in the sidebar"?  `stacktrace` is not well-defined here.

Reply:
In reference to https://github.com/saulpw/visidata/pull/2166#discussion_r1432198312, does this mean for this doc I can mention "Python Exception" or "RuntimeError"?